taas/swagger/swagger.yaml

```diff
@@ -0,0 +1,79 @@
+---
+swagger: "2.0"
+info:
+  description: "This is an implementation of so called Test as a Service. This service\
```

Comment:
How does YAML treat those trailing `\` characters? I think YAML uses `>` and `|` for multi-line strings.

Reply:
https://stackoverflow.com/a/3790497

Reply:
Yup this works, can be checked on 'live UI'. Actually the swagger file was generated by http://editor.swagger.io/ and then I made just minor improvements.